PennMobile/Auth/PennLoginController.swift

```diff
@@ -147,9 +147,11 @@ class PennLoginController: UIViewController, WKUIDelegate, WKNavigationDelegate,
         }
     }
     
+    var handleCancel: (() -> Void)? = nil
```

Comment:
I added a variable in which we can handle the case in which the user decides to simply cancel the login screen.
For my PAC Code vc, I thought it would be ideal if we just make the user go back to the "More" screen if they decide not to login when they should.